index.html

```diff
@@ -11,174 +11,183 @@
 
 <body>
 <header>
-  <div class="tm-logo">
-    <div class="tm-logo-image">
-      <img src="img/logo.svg" />
+  <div class="tm-header tm-container">
```

Comment:
Applied {width: 100% and margin: 0 20px} adds scroll to a page since total with of the block is 100%+40px. White gape at the right on screenshot - http://screen.mindklab.com/incoming/351932d0ece7140caa3fb142c786.png

Reply:
done